doc/examples/reconst_ivim.py

```diff
@@ -0,0 +1,257 @@
+that contains multiple low b-values. The IVIM model can be understood
+as an adaptation of the work of Stejskal and Tanner [Stejskal65]_
+in biological tissue, and was proposed by Le Bihan [LeBihan84]_.
+The model posits that two compartments exist: a slow moving compartment,
+where particles diffuse in a Brownian fashion as a consequence of thermal
+energy, and a fast moving compartment (the vascular compartment), where
+blood moves as a consequence of a pressure gradient. In this second
+compartment, a pseudo diffusion term $\mathbf{D^*}$ is introduced that
+describes the displacement of the blood elements in an assumed randomly
+laid out vascular network, at the macroscopic level. For the perfusion
+to have a physiological meaning, one expects that $\mathbf{D^*}$ is
+greater than $\mathbf{D}$.
+
+The IVIM model expresses the MRI signal as follows:
+
+ .. math::
+    S(b)=S_0(fe^{-bD^*}+(1-f)e^{-bD})
+
+where $\mathbf{b}$ is the gradient value (which is dependent on
+the measurement parameters), $\mathbf{S_{0}}$ is the signal in the absence
+of diffusion gradient sensitization, $\mathbf{f}$ is the perfusion
+fraction, $\mathbf{D}$ is the diffusion coefficient and $\mathbf{D^*}$ is
+the pseudo-diffusion constant, due to vascular contributions.
+
+In the following example we show how to fit the IVIM model
+on a diffusion-weighted dataset and visualize perfusion
+and diffusion. First, we import all relevant modules:
+"""
+
+import matplotlib.pyplot as plt
+
+from dipy.reconst.ivim import IvimModel, ivim_function
+from dipy.data.fetcher import read_ivim
+
+"""
+We get an IVIM dataset using Dipy's data fetcher ``read_ivim``.
+This dataset was acquired with 21 b-values in 3 different directions.
+Volumes corresponding to different directions were registered to each
+other, and averaged across directions. Thus, this dataset has 4 dimensions,
+with the length of the last dimension corresponding to the number
+of b-values.
+"""
+
+img, gtab = read_ivim()
+
+"""
+The variable ``img`` contains a ``nibabel`` NIfTI image object(with the data)
+and gtab contains a GradientTable object(information about
+the gradients e.g. b-values and b-vectors). We get the
+data from img using ``read_data``.
+"""
+
+data = img.get_data()
+print('data.shape (%d, %d, %d, %d)' % data.shape)
+
+"""
+The data has 54 slices, with 256-by-256 voxels in each slice.
+The fourth dimension corresponds to the b-values in the gtab.
+Let us visualize the data by taking a slice midway(z=27) at
+$\mathbf{b} = 0$.
+"""
+
+z = 27
+b = 20
+
+plt.imshow(data[:, :, z, b].T, origin='lower', cmap='gray')
+plt.axhline(y=100)
+plt.axvline(x=170)
+plt.savefig("ivim_data_slice.png")
+plt.close()
+
+"""
+.. figure:: ivim_data_slice.png
+   :align: center
+
+   Heat map of a slice of data
+
+The marked point in the image shows a section containing cerebral spinal
+fluid (CSF) so it should have a very high $\mathbf{f}$ and $\mathbf{D^*}$,
```

Comment:
Got it! You are pointing to the center of a zoom box defined below! Can we add a box on the figure instead of the cross?